account_bank_statement_import_txt_xlsx/models/account_bank_statement_import.py

```diff
@@ -24,13 +24,12 @@ def _get_default_mapping_id(self):
         default=_get_default_mapping_id,
     )
 
-    @api.multi
     def _parse_file(self, data_file):
         self.ensure_one()
         try:
             Parser = self.env["account.bank.statement.import.sheet.parser"]
-            return Parser.parse(self.sheet_mapping_id, data_file, self.filename)
-        except:
+            return Parser.parse(data_file, self.sheet_mapping_id)
+        except Exception:
```

Comment:
`except BaseException:` would be equivalent of `except:`

Reply:
done